maquetado/scss/partials/_styles.scss

```diff
@@ -0,0 +1,6 @@
+@import '_colors';
+
+.container {
+  font-family: 'Roboto', sans-serif;
```

Comment:
I'd move this to a file in the `partials` folder named something like `_fonts` or something that can help you reference different fonts. 🙂